Kickstarter-iOS/Views/Controllers/ThanksViewController.swift

```diff
@@ -280,3 +287,9 @@ internal final class ThanksViewController: UIViewController, UITableViewDelegate
 extension ThanksViewController: ProjectNavigatorDelegate {
   func transitionedToProject(at _: Int) {}
 }
+
+extension ThanksViewController: ThanksCategoryCellDelegate {
+  func thanksSeeAllProjectsTapped(_: ThanksCategoryCell, category: KsApi.Category) {
+    self.viewModel.inputs.categoryCellTapped(category)
```

Comment:
I know this is an existing input but could also be good to align this naming with the delegate method naming once that's updated 🤷‍♂